AcademyApp/Views/LectureDetail/LectureHeaderCollectionViewCell.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  OverviewCollectionViewCell.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 02/07/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+class LectureHeaderCollectionViewCell: UICollectionViewCell, NibLoadableView {
+
+    // MARK: - Stored Propertie
+
+    private var gradientLayer = CAGradientLayer()
+
+    /// ratios from original design
+    private enum UIAspectRatio {
```

Comment:
don't use prefixes when naming stuff, especially Apple prefixes (eg UI, NS, CL, ...)

Reply:
Renamed to AspectRatioConfig.